docs/ref/filters.txt

```diff
@@ -763,8 +763,9 @@ If you wish to sort by non-model fields, you'll need to add custom handling to a
             ]
 
 
-        def filter(self, qs, value):
-            if value in ['relevance', '-relevance']:
-                # sort queryset by relevance
-                return ...
-            return super(CustomOrderingFilter, self).filter(qs, value)
+        def filter(self, qs, values):
```

Comment:
I'd prefer to leave the signature unchanged. Instead of `values` follow with a single line comment: 

     # OrderingFilter is CSV-based, so `value` is a list